docs/recipes/code-coverage.md

```diff
@@ -170,6 +170,28 @@ after_success:
 
 Your coverage report will then appear on coveralls shortly after Travis completes.
 
+## Integrating with AVA's Watch mode
```

Comment:
Reworded the title completely. Like Mark noted, it was not accurate actually, we can't say "AVA's watch" because it's "nodemon's watch". Please review.